### PR TITLE
Pictures app -> Material Design

### PR DESCRIPTION
**Pull request by bdonald25**
@jaypoulz 
@stowellm

### CODE REVIEW

**Comment by stowellm** [comment]
Just show me the app sometime :smile:

File: Apps/Motion/src/edu/uml/cs/isense/motion/dialogs/About.java [hunk 1]
```diff
@@ -9,15 +8,12 @@
...
-	private Button okButton;
-
 	@Override
 	public void onCreate(Bundle savedInstanceBundle) {
 		super.onCreate(savedInstanceBundle);
 		setContentView(R.layout.about);
 
-		okButton = (Button) findViewById(R.id.okButton);
-
+        Button okButton = (Button) findViewById(R.id.okButton);
```
Comment (asalani93) [inline Apps/Motion/src/edu/uml/cs/isense/motion/dialogs/About.java]: indenting

File: Apps/Pictures/res/values/strings.xml [hunk 2]
```diff
@@ -85,9 +86,9 @@ To use the App via the Share Feature in Gallery: While viewing images in your ga
          </string>
   	<string name="About">\n\nThis app can record data regardless of Internet or Wi-Fi connection. After finished recording data, the points you recorded will remain saved. Later, if you have internet connection, you can click on upload (a menu option located at the top of the app). This will show you the session name and time that your data was recorded. Clicking on the upload button will publish that information to iSENSE.\n\n </string>
     <string name="helpText">\nHow to Create a Project for this App:\n\n1. Go to isenseproject.org\n\n2. Login or Register for an Account.\n\n3. Click on Projects.\n\n4. Click on Create Project.\n\n5. Enter a name for your project, then click finish. \n\n6. Enter a Description.\n\n7. When formatting the project, the fields should be as follows:\n\n 
-Name:\t\t\t\t\t\t\t\tType:\n\n        
-Timestamp\t\t\t\t\tTimestamp\n
-Latitude\t\t\t\t\t\t\tLatitude\n
+Name:\t\t\t\t\t\t\t\t\t\tType:\n\n
+Timestamp\t\t\t\tTimestamp\n
+Latitude\t\t\t\t\t\t\t\t\tLatitude\n
```
Comment (asalani93) [inline Apps/Pictures/res/values/strings.xml]: I feel that there may be a better way to do this.  I'm not sure what it is, however.

File: Apps/Pictures/src/edu/uml/cs/isense/riverwalk/Continuous.java [hunk 3]
```diff
@@ -1,29 +1,25 @@
...
+public class Continuous extends ActionBarActivity {
 
-	public static int continuous_interval;
-
-	@SuppressLint("NewApi")
 	@Override
 	protected void onCreate(Bundle savedInstanceState) {
 	super.onCreate(savedInstanceState);
 	setContentView(R.layout.continuous_shooting);
+    getWindow().setLayout(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT);
```
Comment (asalani93) [inline Apps/Pictures/src/edu/uml/cs/isense/riverwalk/Continuous.java]: indeting

**Pull request merged**